tests/ecosystem/upgrade/conftest.py

```diff
@@ -566,3 +653,28 @@ def upgrade_buckets(
         )
 
     return buckets
+
+
+@pytest.fixture(scope='session')
```

Comment:
Just for easier readability - I think it might be better to move it above `mcg_workload_job`. 
Was surprised to see it defined _after_ an invocation :)

Reply:
Ok